src/passes/Print.cpp

```diff
@@ -1865,18 +1880,18 @@ struct PrintSExpression : public OverriddenVisitor<PrintSExpression> {
     o << ')';
   }
   // Module-level visitors
-  void visitFunctionType(FunctionType* curr, Name* internalName = nullptr) {
+  void visitSignature(Signature curr, Name* funcName = nullptr) {
```

Comment:
this looks like a standard visitor, but it isn't, I think? maybe another name would make it less likely to confuse. like `handleSignature`?